virttest/utils_net.py

```diff
@@ -3744,3 +3746,190 @@ def map_hostname_ipaddress(hostname_ip_dict, session=None):
                 return False
     logging.info("All the hostnames and IPs are mapped in %s", hosts_file)
     return True
+
+
+def _remove_file(session, file_path):
+    """
+    Remove a file at file_path
+
+    :param file_path: file path to remove
+    :param session: a logined session
+    """
+
+    clean_cmd = "del %s" % file_path
+    session.cmd_status(clean_cmd)
+
+
+def _copy_file(session, src_path, dst_folder="c:\\"):
+    """
+    Copy a file at src_path to dst_folder.
+    If a same name file already exists at dst_folder, it will be replaced.
+
+    :param session: a logined sesison
+    :param src_path: file to copy from
+    :param dst_folder: folder to copy to
+
+    """
+
+    copy_cmd = "xcopy %s %s /y" % (src_path, dst_folder)
+    session.cmd_status(copy_cmd)
+
+
+def _get_traceview_path(session, params):
+    """
+    :param session: a session to send cmd
+    :param params: the test params
+    :return: the proper traceview path
+    """
+
+    traceview_path_template = "WIN_UTILS:\\traceview\\%s\\%s\\traceview.exe"
+    # traceview_ver = params["traceview_version"]
+    traceview_ver = "win10"
+    os_version = system.version(session)
+    main_ver = int(os_version.split('.')[0])
+    if main_ver < 10:
+        traceview_ver = "win8"
+    os_arch = "%PROCESSOR_ARCHITECTURE%"
```

Comment:
Seems this is a fixed string, unnecessary to have a `%s` for it in `traceview_path_template`.